source/mapping-rules/fields/data-type-conversion-guide.txt

```diff
@@ -0,0 +1,229 @@
+.. _rm-data-type-conversion-guide:
+
+==========================
+Data Type Conversion Guide
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page outlines examples that show the result of converting the 
+source value and data-type to a specific BSON data-type using customized 
+fields.
+
+Behavior
+~~~~~~~~
```

Comment:
(suggestion)
What about converting this whole section into a three item bulleted list?